### PR TITLE
Add csv export task for Support::Cases

### PR DESCRIPTION
**Pull request by kennyevil**
## Changes in this PR

- add a `to_csv` method into `Support::Case`
- add case data as an option to the `export` rake task

### CODE REVIEW

**Comment by peterdavidhamilton** [comment]
[Manual export of service activity data](https://trello.com/c/ZuoGbMQg/107-manual-export-of-service-activity-data)

**Review by AndrewCowle — COMMENTED** [review]
What do you guys think about having this as a concern (likely that we might want to have CSV export on a few models) @peterdavidhamilton @kennyevil ?

**Comment by kennyevil** [comment]
> What do you guys think about having this as a concern (likely that we might want to have CSV export on a few models) @peterdavidhamilton @kennyevil ?

This is a good idea. I've moved it into a concern called csvable.

**Comment by peterdavidhamilton** [comment]
> > What do you guys think about having this as a concern (likely that we might want to have CSV export on a few models) @peterdavidhamilton @kennyevil ?
> 
> This is a good idea. I've moved it into a concern called csvable on #672

For the reasons discussed it might be easier not to. I'm not fussed though but we don't want concerns being shared across the namespace, which means we still double up, and the hope is we deleted this anyway. Are we ok to revert that?

**Review by peterdavidhamilton — APPROVED** [review]

File: lib/tasks/export.rake [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+desc "Export ActivityLog items"
+task export: :environment do
+  file = Rails.root.join("public/activity_log.csv")
+  data = ActivityLogItem.to_csv
+
+  File.open(file, "w+") { |f| f.write(data) }
+end
+
 namespace :support do
   namespace :cases do
```
Comment (peterdavidhamilton) [inline lib/tasks/export.rake:11]: There are potentially two tasks in the support namespace, one for exporting cases, the other for log items. This one is named wrong.

File: lib/tasks/export.rake [hunk 2]
```diff
@@ -5,3 +5,15 @@ task export: :environment do
 
   File.open(file, "w+") { |f| f.write(data) }
 end
+
+namespace :export do
```
Comment (peterdavidhamilton) [inline lib/tasks/export.rake:9]: There is a `case_management` namespace in use within our tasks

**Review by d-a-v-e — APPROVED** [review]

**Pull request merged**